lib/utilities/map_styles/create_game_map.dart

```diff
@@ -0,0 +1,160 @@
+String createGameMapStyle = '''[
```

Comment:
copied and pasted json to make the map greyscale

Reply:
oh sweet! I've been planning to look up to see how to do this!